src/home_robot/home_robot/mapping/semantic/categorical_2d_semantic_map_module.py

```diff
@@ -67,13 +82,15 @@ def __init__(
              consider it as explored
             map_pred_threshold: number of depth points to be in bin to
              consider it as obstacle
+            must_explore_close: reduce the distance we need to get to things to make them work
```

Comment:
@cpaxton Let's put this in the config?

Reply:
I'm not so familiar with these configs, what do i need to do in order to do this?

Reply:
Can we add this as an issue and tag me? I dont want to make these kinds of changes without hardware tests